### ISSUE TITLE
Overpowered Food

### ISSUE THREAD

**Issue by Dinrae** [itsmeow/betteranimalsplus#310]
**Overpowered Food**
This mod is great, but it breaks food balance, because, for example, fried egg is overpowered, giving almost full saturation!

**Rebalance Food**
Take example on other food mods, like Valleycraft or Farming Delight, and rebalance food stats.

Or allow us to disable food in config please.

**Comment by purejosh** [comment]
Yes, this egg needs to be nerfed. It's out of control. Simply creating a chicken farm provides unlimited, overpowered food. And we cannot use a datapack to disable the recipe, since it will disable all fried egg recipes from other mods that are far more balanced than this one.
